build.sh

```diff
@@ -1,7 +1,10 @@
-﻿nuget restore
+﻿
+nuget restore
+
 find Samples -type f -name "*.csproj" | for proj in $(cat) ; do 
 msbuild $proj /p:Configuration=Release
-done
-find Samples -type f -name "*.csproj" | for proj in $(cat) ; do 
-msbuild $proj /p:Configuration=Debug
-done
+done || exit 1
+
+# find Samples -type f -name "*.csproj" | for proj in $(cat) ; do 
```

Comment:
@cameron-morrow-toptal are these build changes intentional?

Reply:
yes, I disabled the debug build because it wasn't necessary for building the nuget package. I'll probably add it to a separate script in the future for CI